cmd/distro.go

```diff
@@ -678,3 +685,72 @@ func NewDistroReportCmd() (*cobra.Command, error) {
 	distroReportCmd.Flags().String("name", "", "the distro name")
 	return distroReportCmd, nil
 }
+
+func NewDistroExportCmd() (*cobra.Command, error) {
+	distroExportCmd := &cobra.Command{
+		Use:   "export",
+		Short: "export distributions",
+		Long:  `Export distributions.`,
```

Comment:
Maybe add the formats you can export it as

Reply:
I will add them to the export flag.